op-service/txmgr/txmgr.go

```diff
@@ -583,18 +575,25 @@ func (m *SimpleTxManager) publishTx(ctx context.Context, tx *types.Transaction,
 		case errStringMatch(err, txpool.ErrReplaceUnderpriced):
 			l.Warn("transaction replacement is underpriced", "err", err)
 			m.metr.TxPublished("tx_replacement_underpriced")
-			continue // retry with fee bump
+			// retry tx with fee bump, unless we already just tried to bump them
+			if !sendState.bumpFees {
+				sendState.bumpFees = true
+				continue
+			}
```

Comment:
hmm, wouldn't we like to continue fee bumping? it may be the case, e.g. after a restart, to bump several times before the replacement error goes away and we bumped high enough. If we now return with failure after a single bump, we would never bump far away.

```suggestion
			sendState.bumpFees = true
			continue
```

Reply:
This is what it already should do. The change you suggest would result in an endless loop once we hit the point where fees can't be bumped any further.

Reply:
I see, this is within `publishTx`, which is called in a loop inside `sendTx`, which would case the fee bumping, right? So this only avoid the endless loop inside a single `publishTx` call?